### PR TITLE
improve dockerfile

### LINKED ISSUE #89: [BUG] Docker file should use the corrert source
Right now, we should NOT use the zip release to create dockers, but the very source that is checked out during the action

Affected files are
```
.github/workflows/docker-build.yml
.github/workflows/docker-tag-release.yml
```

### PR DESCRIPTION
**Pull request by alegrey91**
Closes #89

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ ARG CRYPTOGRAPHY_VERSION=3.3.2
 ARG TERRAFORM_VERSION=0.12.0
 ARG TERRAFORM_PROVIDER_VERSION=0.6.2
 ARG TERRAFORM_PROVIDER_RELEASE=0.6.2+git.1585292411.8cbe9ad0
-ARG TERRIBLE_VERSION=1.1.1
+#ARG TERRIBLE_VERSION=1.1.1
```
Comment (89luca89) [inline Dockerfile:11]: delete this

File: README.md [hunk 2]
```diff
@@ -620,12 +620,12 @@ docker run \
 
 **Notes:**
 
-* The volume `/var/run/libvirt/libvirt-sock` is mandatory if you want to run Terrible locally (a local QEMU/KVM instance). In this way you directly interact with the QEMU/KVM api, provided by the system.
+* The volume `/var/run/libvirt/libvirt-sock` is mandatory if you want to run Terrible locally (a local QEMU/KVM instance). In this way you directly interact with the QEMU/KVM api, provided by the system (required).
```
Comment (89luca89) [inline README.md:623]: It's not required if the Libvirt server is remote

File: README.md [hunk 3]
```diff
@@ -620,12 +620,12 @@ docker run \
...
 
-* The volume `./inventory-test.yml` is to include the inventory inside the container, to deploy the infrastructure.
+* The volume `./inventory-test.yml` is to include the inventory inside the container, to deploy the infrastructure (optional).
```
Comment (89luca89) [inline README.md:625]: this instead is required, without inventory how does this work?

File: README.md [hunk 4]
```diff
@@ -620,12 +620,12 @@ docker run \
...
 
-* The volume `~/VirtualMachines/` is to include the `qcow2` images inside the container to deploy them.
-* The volume `~/.ssh` is to include your ssh keys into the container and deploy them inside the infrastructure machines.
+* The volume `~/path/to/vm/images/` is to include the `qcow2` images (previously generated with Packer) inside the container to deploy them (required).
```
Comment (89luca89) [inline README.md:627]: required only if local Libvirt

File: README.md [hunk 5]
```diff
@@ -620,12 +620,12 @@ docker run \
...
+* The volume `~/.ssh` is to include your ssh keys into the container and deploy them inside the infrastructure machines (optional).
```
Comment (89luca89) [inline README.md:628]: Required if an ssh_key is declared in the inventory

**Review by 89luca89 — CHANGES_REQUESTED** [review]

**Review by J4CKR3D — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (alegrey91) [reply README.md:625]: you can also write it inside the container :)

File: README.md (hunk 5 above)
Comment (alegrey91) [reply README.md:628]: you can generate ssh keys inside container for a one-time use

File: README.md (hunk 3 above)
Comment (89luca89) [reply README.md]: Not really the type of use we're advertising here isn't it?

File: README.md (hunk 5 above)
Comment (89luca89) [reply README.md]: Not right now, but not the type of use this is aiming for isn't it?

**Review by 89luca89 — APPROVED** [review]

**Pull request merged**